### PR TITLE
Create header

### PR DESCRIPTION
**Pull request by KirilushkinAleksandr**
Page header added

### CODE REVIEW

File: client/src/components/header/header.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import './header.scss';
+
+class Header extends React.Component {
```
Comment (EasileleM) [inline client/src/components/header/header.js]: split blocks below into separate components

File: client/src/components/header/contacts.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        <address className="header__wrapper">
+          <span className="header__text header__text_md header__text_email">
+            info@shopy.com
+            </span>
+          <span className="header__text header__text_md header__text_phone">
+            453 - 5553 - 996
+            </span>
+        </address>
+        <nav className="header__wrapper">
+          <div className="header__icon header__icon_facebook">
```
Comment (miffn3) [inline client/src/components/header/contacts.js]: why div here?

File: client/src/components/header/navigation.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            <input className="header__search-button" type="checkbox" id="search-button">
+            </input>
+            <input className="header__search-bar" type="text">
+            </input>
+            <label className="header__icon header__icon_search" htmlFor="search-button">
+            </label>
+          </form>
+          <a className="header__icon header__icon_big header__icon_user" href="https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md">
+          </a>
+          <a className="header__icon header__icon_big header__icon_cart" href="https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md">
```
Comment (EasileleM) [inline client/src/components/header/navigation.js]: It is not sufficient solution. Given mock provides item in bucket counter functionality.

File: client/src/components/header/header.scss [hunk 4]
```diff
@@ -0,0 +1,177 @@
+@import url(//db.onlinewebfonts.com/c/32eb6c28879dcfcaf97b75a5194441e9?family=HarmoniaSansProCyrRegular);
```
Comment (EasileleM) [inline client/src/components/header/header.scss]: This fonts are prohibited in heroku in terms of security (not https)

File: client/src/components/header/header.scss [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+    &_user {
+      background: url(../../assets/User_Ico.png) no-repeat right;
+    }
+
+    &_cart {
+      background: url(../../assets/Cart_Ico.png) no-repeat right;
+    }
+  }
+
+  &__logo {
```
Comment (EasileleM) [inline client/src/components/header/header.scss]: We should use text here instead image

**Pull request merged**